Lib/stdlib_deprecations.py

```diff
@@ -0,0 +1,182 @@
+import dataclasses
+import re
+
+
+@dataclasses.dataclass(slots=True)
```

Comment:
```suggestion
@dataclasses.dataclass(slots=True, frozen=True)
```

I think it is meant to be immutable.